homeassistant/components/zwave_js/api.py

```diff
@@ -263,6 +264,47 @@ def node_removed(event: dict) -> None:
     )
 
 
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/get_config_parameters",
+        vol.Required(ENTRY_ID): str,
+        vol.Required(NODE_ID): int,
+    }
+)
+@callback
+def websocket_get_config_parameters(
+    hass: HomeAssistant, connection: ActiveConnection, msg: dict
+) -> None:
+    """Get a list of configuration parameterss for a Z-Wave node."""
+    entry_id = msg[ENTRY_ID]
+    node_id = msg[NODE_ID]
+    client = hass.data[DOMAIN][entry_id][DATA_CLIENT]
+    node = client.driver.controller.nodes[node_id]
+    values = node.get_configuration_values()
+    result = {}
+    for value_id, zwave_value in values.items():
+        metadata = zwave_value.metadata
+        result[value_id] = {
+            "property": zwave_value.property_,
+            "configuration_value_type": zwave_value.configuration_value_type.value,
+            "metadata": {
+                "description": metadata.description,
+                "label": metadata.label,
+                "type": metadata.type,
+                "min": metadata.min,
+                "max": metadata.max,
+                "unit": metadata.unit,
+                "writeable": metadata.writeable,
```

Comment:
might be good to add the `readable` property too. I've noticed some config parameters have that set to false. Example: https://github.com/home-assistant/core/blob/dev/tests/fixtures/zwave_js/bulb_6_multi_color_state.json#L514